src/core/ucc_lib.c

```diff
@@ -393,6 +393,9 @@ ucc_status_t ucc_finalize(ucc_lib_info_t *lib)
     /* If some CL components fails in finalize we will return
```

Comment:
seems this comment is incorrect as we always returning UCC_OK

Reply:
yes, comment is incorrect. Finalize function doesn't return any value because its type is ucs_class_cleanup_func_t.
Removed the comment.